ArcSlider/ArcSliderBehaviorDecorator.js

```diff
@@ -180,7 +190,12 @@ const ArcSliderBehaviorDecorator = hoc((config, Wrapped) => {
 
 			const angle = positionToAngle(coordsInSvg, radius * 2 - strokeWidth);
 			// get the value based on the angle, min and max
-			let value = angleToValue(angle, min, max, startAngle, endAngle);
+			let value;
+			if (max <= min) {
+				value = angleToValue(angle, min, min, startAngle, endAngle);
+			} else {
+				value = angleToValue(angle, min, max, startAngle, endAngle);
+			}
```

Comment:
```suggestion
			const value = angleToValue(angle, min, Math.max(min, max), startAngle, endAngle);
```
How about it? There may be other good alternatives.

Reply:
yes, it's better this way, but I will need to use "let" because the value is updated after this